docs/reports/peer_testing_2/usability_tasks.md

```diff
@@ -0,0 +1,117 @@
+    valuable if you are familiar with the language)
+    2. Choose a voice for that language
+    3. Convert your input to a .wav and .txt file
+    4. Inform the examiner which avatar you'd like to use: Barb or Boy
+
+2. Let the examiner do the back-end work that would happen if the API was integrated
+    a. Send the .wav file, .txt file, language, and avatar selection to the API
+    b. Wait for the API to generate the video
+
+3. Watch the video
+
+Repeat this scenario with a different language each time.
+
+### User Scenario 2: Voice Test
+
+1. On the Kukarella web app
+
+    1. Enter text in a language
+    2. Select a voice for that language (try a different one each time you repeat this scenario)
+    3. Convert your input to a .wav and .txt file
+    4. Inform the examiner which avatar you'd like to use: Barb or Boy
+
+2. Let the examiner do the back-end work that would happen if the API was integrated
+    a. Send the .wav file, .txt file, language, and avatar selection to the API
+    b. Wait for the API to generate the video
+
+3. Watch the video
+
+Repeat this scenario with a different voice each time.
+
+### User Scenario 3: Voice Effects
+
+1. On the Kukarella web app
+
+    1. Enter text in a language
+    2. Select a voice for that language that supports effects
+    3. Play around with the following effects to see if the animated avatar looks natural with each:
+        - Pitch
+        - Pause
+        - Speed
+        - Say As
+        - Emphasis
+        - Volume
+        - Mute
+        - Breath
+        - Whisper
+    4. Convert your input to a .wav and .txt file
+    5. Inform the examiner which avatar you'd like to use: Barb or Boy
+
+2. Let the examiner do the back-end work that would happen if the API was integrated
+    a. Send the .wav file, .txt file, language, and avatar selection to the API
+    b. Wait for the API to generate the video
+
+3. Watch the video
+
+Repeat this scenario with different effects each time.
+
+### User Scenario 4: Mix and Match
+
+1. On the Kukarella web app
+
+    1. Enter text in a language (you may use another language; your feedback will be more valuable if you are familiar
+     with the language)
+    2. Select a voice for that supports effects (try a language that doesn't match the text you entered)
+    3. Play around with the following effects:
+        - Pitch
+        - Pause
+        - Speed
+        - Say As
+        - Emphasis
+        - Volume
+        - Mute
+        - Breath
+        - Whisper
+    4. Convert your input to a .wav and .txt file
+    5. Inform the examiner which avatar you'd like to use: Barb or Boy
+
+2. Let the examiner do the back-end work that would happen if the API was integrated
+    a. Send the .wav file, .txt file, language, and avatar selection to the API
+    b. Wait for the API to generate the video
+
+3. Watch the video
+
+Repeat this scenario with different parameters each time for the text, language, voice, and effects.
```

Comment:
I almost feel like this is the only scenario we need? We could just repeat this with different parameters each time?

Reply:
I think it's valuable to have the other scenarios that isolate the changes so that we know if there are specific issues with each change individually. This last one is to see if there's a combination that's really awkward.